pkg/netutil/netutil.go

```diff
@@ -39,3 +42,42 @@ func IsPortActive(port string) bool {
 	util.Warning("Unable to properly check port status: %v", oe)
 	return false
 }
+
+// IsLocalIP returns true if the provided IP address is
+// assigned to the local computer
+func IsLocalIP(ipAddress string) bool {
+	localIPs, err := GetLocalIPs()
+	if err != nil {
+		fmt.Printf("Error fetching local IPs: %v\n", err)
+		return false
+	}
```

Comment:
```suggestion
	dockerIP, err := dockerutil.GetDockerIP()
	
	if err != nil {
		util.Warning("Failed to get Docker IP address: %v", err)
		return false
	}
	
	if ipAddress == dockerIP {
		return true
	}
	
	localIPs, err := GetLocalIPs()

	if err != nil {
		util.Warning("Error fetching local IPs: %v", err)
		return false
	}
```

Reply:
This is also a tiny bit better performance-wise, thanks!